workers/tester.sh

```diff
@@ -4,7 +4,7 @@
 
 # This script should always run as if it were being called from
 # the directory it lives in.
-script_directory=`dirname "${BASH_SOURCE[0]}"`
+script_directory=`cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd`
```

Comment:
No I don't believe it is. The first `cd` command is run in a subshell so once that subshell exits the current shell hasn't changed directories. To prove this:

```
kurt@moss-greenelab:~/Development/data_refinery/workers$ echo `cd .. && pwd` && pwd
/home/kurt/Development/data_refinery
/home/kurt/Development/data_refinery/workers
```